src/development/going-live/multi-app.md

```diff
@@ -0,0 +1,75 @@
+# Serving multiple applications
```

Comment:
This page is still incomplete, pending it being way easier to add multiple domains in the first place.  Feedback still welcome but it's likely buggy.